pkgs/applications/networking/cluster/fluxcd/default.nix

```diff
@@ -50,6 +51,8 @@ buildGoModule rec {
     done
   '';
 
+  passthru.updateScript = ./update.sh;
```

Comment:
Does nix-update-script not work here?

Reply:
Doesn't handle the extra manifest. Or does it?

Reply:
Ah right, I would guess not.